.github/SUPPORT.md

```diff
@@ -1,28 +1,22 @@
-# Support #
+[![Zulip][zulip-badge]][zulip-link]
+[![Discourse][discourse-badge]][discourse-link]
+
+[discourse-badge]: https://img.shields.io/badge/Discourse-forum-informational.svg
+[discourse-link]: https://coq.discourse.group/
 
-Get in touch with the user community and ask questions about Coq on
-our [Discourse forum][]. Posts in other languages than English are
-explicitly welcome there. There is also a historic mailing list called
-the [Coq-Club][] which has lots of subscribers, and an IRC channel
-(`irc://irc.freenode.net/#coq`).
+[zulip-badge]: https://img.shields.io/badge/Zulip-chat-informational.svg
+[zulip-link]: https://coq.zulipchat.com/
+
+# Support #
 
-In addition, you may also ask questions about Coq on [Stack
-Overflow][] (use the tag [coq][Stack Overflow tag]) or on the
-meta-theory of Coq on the [TCS Stack Exchange][] (which also has a
-[coq][TCS SE tag] tag).
+<!-- content copied verbatim from "Questions and discussion" in README.md -->
 
-You can reach the Coq development team through the [development
-category][] of the above mentioned Discourse forum, the [Gitter
-channel][], and of course the bug tracker.
+We have a number of channels to reach the user community and the
+development team:
 
-See also [coq.inria.fr/community](https://coq.inria.fr/community.html).
+- Our [Zulip chat][zulip-link], for casual and high traffic discussions.
+- Our [Discourse forum][discourse-link], for more structured and easily browsable discussions and Q&A.
+- Our historical mailing list, the [Coq-Club](https://sympa.inria.fr/sympa/info/coq-club).
```

Comment:
Poor coq-club, it's "historical" now

Reply:
Yep, we cannot decently bury it given the number of users, but we must have a consistent communication regarding our privileged platforms IMHO.

Reply:
I hadn't realized up to this point that discourse was now preferred.